src/java.compiler/share/classes/javax/lang/model/util/AbstractTypeVisitor6.java

```diff
@@ -138,15 +139,15 @@ public R visitIntersection(IntersectionType t, P p) {
     }
 
     /**
-     * {@inheritDoc}
+     * {@inheritDoc TypeVisitor}
      *
      * @implSpec The default implementation of this method in {@code
      * AbstractTypeVisitor6} will always throw {@code
      * new UnknownTypeException(t, p)}.  This behavior is not required of a
      * subclass.
      *
-     * @param t  {@inheritDoc}
-     * @param p  {@inheritDoc}
+     * @param t  {@inheritDoc TypeVisitor}
+     * @param p  {@inheritDoc TypeVisitor}
      * @return a visitor-specified result
      * @throws UnknownTypeException
```

Comment:
Similar comment about inheriting exception info.

Reply:
Good idea; I included the annotation value variant too -- I'll update the CSR for the change in annotation value visitor class.